metaflow/plugins/argo/argo_workflows_cli.py

```diff
@@ -291,17 +291,23 @@ def create(
                 "due to Kubernetes naming conventions\non Argo Workflows. The "
                 "original flow name is stored in the workflow annotation.\n"
             )
-
-        if ARGO_WORKFLOWS_UI_URL:
-            obj.echo("See the deployed workflow here:", bold=True)
-            argo_workflowtemplate_link = "%s/workflow-templates/%s" % (
-                ARGO_WORKFLOWS_UI_URL.rstrip("/"),
-                KUBERNETES_NAMESPACE,
-            )
+        try:
+            if ARGO_WORKFLOWS_UI_URL:
+                workflow_template_url_values = {
+                    "uri": "workflow-templates",
+                    "namespace": KUBERNETES_NAMESPACE,
+                    "workflow_name": obj.workflow_name,
+                    "suffix": obj.workflow_name,
+                }
+                obj.echo("See the deployed workflow here:", bold=True)
+                argo_workflowtemplate_link = ARGO_WORKFLOWS_UI_URL.rstrip("/").format(
+                    **workflow_template_url_values
+                )
+        except:
             obj.echo(
-                "%s/%s\n\n" % (argo_workflowtemplate_link, obj.workflow_name),
-                indent=True,
+                "workflow template was created - errored in formatting workflow template URL"
```

Comment:
error message could be more specific that `METAFLOW_ARGO_WORKFLOWS_UI_URL` could not be used to format the url.

This exception is never raised though, if you try it out with a template url that is missing the required keys
```
METAFLOW_ARGO_WORKFLOWS_UI_URL=http://example.com/
```

the only exception you will get is if the template string contains keys that are not present in `workflow_template_url_values`, such as
```
METAFLOW_ARGO_WORKFLOWS_UI_URL=http://example.com/{not_present}
```

Reply:
Yes that is the intended behavior. Since it's an existing config var in an effort to be backwards compatible this was a little neat solution